lib/zigler/zig.ex

```diff
@@ -234,6 +234,8 @@ defmodule Zigler.Zig do
 
       Logger.info("downloading zig version #{version} and caching in #{@zig_dir_path}.")
       download_location = "https://ziglang.org/download/#{version}/#{archive}"
+      download_location = "https://ziglang.org/builds/zig-freebsd-x86_64-0.7.0+d4c167f3c.tar.xz"
```

Comment:
maybe we could pass in a custom URL directly from the end project's `config.exs` ? The first part https://ziglang.org/builds/ is the variant we're interested in, and keep the nice zigler auto detection.

Reply:
I would definitely like to support pulling from master in *some form* but I don't generally have the time to maintain this except after major releases of zig (my stance on this may change after 1.0.0)

Reply:
sure that's very fair. I don't see that as maintaining support, but making it easier to hack on it. i.e. "caveat coder" the onus on the user to make it work.